alacritty/src/main.rs

```diff
@@ -89,7 +89,14 @@ fn main() -> Result<(), Box<dyn Error>> {
 
 /// `msg` subcommand entrypoint.
 #[cfg(unix)]
-fn msg(options: MessageOptions) -> Result<(), Box<dyn Error>> {
+#[allow(unused_mut)]
+fn msg(mut options: MessageOptions) -> Result<(), Box<dyn Error>> {
+    #[cfg(not(any(target_os = "macos", windows)))]
+    if let cli::SocketMessage::CreateWindow(window_options) = &mut options.message {
+        window_options.activation_token = std::env::var("XDG_ACTIVATION_TOKEN")
+            .or_else(|_| std::env::var("DESKTOP_STARTUP_ID"))
+            .ok();
+    }
```

Comment:
Can probably add an import for `cli::SocketMessage` and `std::env` here.

Reply:
The thing is that it's all deep in `cfg`, so you'd need a lot of `use` under cfg and it's just way more verbose for what it is.

Reply:
We still want the imports at the top because that's the only way to notice when the `cfg` is not necessary anymore. Anything else will just cause inconsistencies when in the future someone comes along and needs these imports for other locations in this file.